spark/dl/pom.xml

```diff
@@ -158,6 +158,41 @@
                 <artifactId>maven-shade-plugin</artifactId>
                 <version>3.0.0</version>
                 <configuration>
+                    <createDependencyReducedPom>false</createDependencyReducedPom>
+                    <artifactSet>
+                        <excludes>
+                            <exclude>org.apache.opennlp:opennlp-tools</exclude>
```

Comment:
So if anyone want to add a new dependency, she/he need to add an exclude line. I think this list is not easy to maintain..